### DISCUSSION TITLE
Verse; is it applicable to be added to linguist?

### DISCUSSION THREAD

**Discussion by Atayeu** [github-linguist/linguist#7167] [General]
**Verse** is a relatively newer programming language created by Epic Games to be used in Unreal Engine and UEFN for game design.

Is the language able to be scanned if linguist is supplied some .verse files to create syntax highlighting in Github? 

I read through the, how to add a language, and such documentation but was a bit over my head...

**Comment by lildude** [comment]
It doesn't meet our usage requirements just yet, but it's not far off so it might do by the time the next release is made around the end of March '25.

If you don't have the confidence to add support yourself, you can [open an issue](https://github.com/github-linguist/linguist/issues/new?template=new_language.md) and wait for someone else in the Verse community to submit a PR.

**Reply by xyvker-septix-fEkdy8** [reply]
This
